src/renderer/pages/TorrentListPage.js

```diff
@@ -220,6 +196,29 @@ module.exports = class TorrentList extends React.Component {
           {downloadIcon}
         </i>
       )
+
+      if (TorrentPlayer.isPlayableTorrentSummary(torrentSummary)) {
+        playButton = (
+          <i
+            key='play-button'
+            title={playTooltip}
+            className={'button-round icon play ' + playClass}
```

Comment:
Just realized there's a bug here. `playClass` is used here, but it's defined later, on line 220. So it gets set to `undefined` which is causing a rendering a bug.

<img width="132" alt="screen shot 2016-08-23 at 5 23 49 pm" src="https://cloud.githubusercontent.com/assets/121766/17914367/5fd68b5a-6956-11e6-9aaf-43a020b957b5.png">

The radial is getting rendered *under* the play button instead of over it.